services/server/config/config.go

```diff
@@ -55,6 +55,8 @@ type Config struct {
 	Cgroup CgroupConfig `toml:"cgroup"`
 	// ProxyPlugins configures plugins which are communicated to over GRPC
 	ProxyPlugins map[string]ProxyPlugin `toml:"proxy_plugins"`
+	// Timeouts specified in seconds
+	Timeouts map[string]int `toml:"timeouts"`
```

Comment:
int -> float?

int is fine for the current shim stuff, but in future, we might want to add something that wants the timeout value to be less than 1s.

Reply:
or maybe `time.ParseDuration` string

Reply:
Updated